l10n_it_fatturapa_in/models/attachment.py

```diff
@@ -57,6 +57,11 @@ class FatturaPAAttachmentIn(models.Model):
 
```

Comment:
Come mai questa PR di migrazione per **l10n_it_fatturapa_in_rc** modifica anche **l10n_it_fatturapa_in**?

Reply:
Probabilmente qualche aggiornamento del branch non voluto.
Rimuovo queste modifiche

Reply:
sono modifiche introdotte da questo commit non nostro: 647af23885a8a672f9827ff094f3b658c348c747

Reply:
👍🏻

Reply:
👍🏻

Reply:
```
git checkout 16.0
git pull
git checkout 16.0-mig-l10n_it_fatturapa_in_rc
git rebase 16.0
# Controlli che sia tutto ok
git push -f
```

Reply:
senza git checkout, credo basterebbe:

```
git fetch # o git fetch <remote name> 16.0 se lavori con più remote
git rebase 16.0
git push -f 
```

Reply:
> senza git checkout, credo basterebbe:
> 
> ```
> git fetch # o git fetch <remote name> 16.0 se lavori con più remote
> git rebase 16.0
> git push -f 
> ```

A me `16.0` non si aggiorna semplicemente con un `fetch`, ma rimane indietro rispetto a `origin/16.0`; non dovrebbe essere invece `git rebase origin/16.0`? Supponendo che `origin` sia il remote di OCA

Reply:
Hai ragione sul rebase, avevo dimenticato un pezzo nel comando :sweat: 

Ma se specifichi `git rebase origin/16.0` (invece che solo `git rebase 16.0`) fa il rebase in base all'ultimo stato noto del branch sul remote (che non è detto sia lo stesso stato del tuo branch locale 16.0). `git pull` fa semplicemente `git fetch` + `git merge` del branch remoto sul branch corrente.

Scusa, adesso mi zittisco. Non era importante, ma git mi piace molto haha.

Reply:
> Ma se specifichi `git rebase origin/16.0` (invece che solo `git rebase 16.0`) fa il rebase in base all'ultimo stato noto del branch sul remote (che non è detto sia lo stesso stato del tuo branch locale 16.0). `git pull` fa semplicemente `git fetch` + `git merge` del branch remoto sul branch corrente.

Sì, infatti proponevo la modifica ai tuoi step dove avevi fatto subito prima `git fetch`; di solito uso la procedura con `checkout` così colgo l'occasione per aggiornare anche `16.0` in locale.

> Scusa, adesso mi zittisco. Non era importante, ma git mi piace molto haha.

Ma va, è sempre utile un po' di confronto :smile:

Reply:
@SirAionTech @aleuffre grazie per i suggerimenti come sempre
Purtroppo però faccio prima a rifare siccome il rebase mi da dei conflitti...
Aggiorno subito